src/tools/perf/perftest.c

```diff
@@ -228,10 +228,10 @@ static void sock_rte_barrier(void *rte_group, void (*progress)(void *arg),
         unsigned snc;
 
         snc = magic;
-        safe_send(group->connfd, &snc, sizeof(unsigned), progress, arg);
+        safe_send(group->connfds[0], &snc, sizeof(unsigned), progress, arg);
```

Comment:
`group->connfds[0]` (and `[1]`) is used in several places. Maybe introduce a function/macro to access the send/recv fd of a `group`? Or use `(sendfd, recvfd)` as Yossi suggested.